modules/git/tests/repos/repo_lfs/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/master
```

Comment:
I don't understand why it needs a new git repo for testing purpose. There are already plenty.

Reply:
This is for lfs test. There is no test repository for lfs under `git/tests`.

Reply:
Why it must be under `git/tests`?

Reply:
removed at 87e12a1df9